panther_analysis_tool/util.py

```diff
@@ -138,4 +138,4 @@ def deep_get(obj: Dict, path: List[str], default: Any = None) -> Any:
 
 
 def to_list(listish: Any) -> List:
-    return listish if hasattr(listish, "__iter__") else [listish]
+    return listish if isinstance(listish, list) else [listish]
```

Comment:
@maxrichie5 does this need to handle other list-ish things (`set`, `tuple`)?

Reply:
(that'd be `isinstance(listish, tuple)` and `isinstance(listish, set)` IIRC), but of course tests should cover those too.

Reply:
I can add those to the tests. It should not. It is only supposed to make a single a list and keep a list a list

Reply:
Gotcha. I'd like to see a test to document that behavior explicitly, but I'll provisionally approve the PR — just merge once you have that done, please.

Reply:
@maxrichie5 yes! :shipit: